### ISSUE TITLE
EyeOnWater data: list index out of range

### ISSUE THREAD

**Issue by iskiselev** [kdeyev/eyeonwater#95]
I see many instances of next error in my log from integration:
```
Logger: custom_components.eyeonwater
Source: helpers/update_coordinator.py:313
Integration: EyeOnWater ([documentation](https://github.com/kdeyev/eyeonwater))
First occurred: 3:19:06 PM (21 occurrences)
Last logged: 8:19:27 PM

Unexpected error fetching EyeOnWater data: list index out of range
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 313, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 269, in _async_update_data
    return await self.update_method()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/eyeonwater/__init__.py", line 59, in async_update_data
    await eye_on_water_data.read_meters(days_to_load=3)
  File "/config/custom_components/eyeonwater/coordinator.py", line 43, in read_meters
    await meter.read_historical_data(
  File "/usr/local/lib/python3.12/site-packages/pyonwater/meter.py", line 78, in read_historical_data
    elif historical_data[-1].reading == self.last_historical_data[
         ~~~~~~~~~~~~~~~^^^^
IndexError: list index out of range
```

I don't see any usage data yet, but in my case meter have not reported data to the site for about a month. It just started reporting data again - but still no data in ha.

**Comment by kdeyev** [comment]
Thank you for reporting the issue.
I've release the blind fix in [v2.5.2](https://github.com/kdeyev/eyeonwater/releases/tag/v2.5.2)

**Comment by iskiselev** [comment]
Thank you, it works with the new version!
